git_services/git_services/init/cloner.py

```diff
@@ -63,9 +65,9 @@ def _initialize_repo(self):
         self.cli.git_init()
         # NOTE: For anonymous sessions email and name are not known for the user
         if self.user.email is not None:
-            self.cli.git_config(f"user.email {self.user.email}")
+            self.cli.git_config(f"user.email '{self.user.email}'")
         if self.user.full_name is not None:
-            self.cli.git_config(f"user.name {self.user.full_name}")
+            self.cli.git_config(f"user.name '{self.user.full_name}'")
```

Comment:
Should these be escaped double ticks instead? Not sure

Reply:
I dont think so. I already tested it and it works. And the ticks do not show up in the git config. I think double ticks vs single only matters if you want the shell to substitute a variable in there and then you would use double ticks. But that is not the case here.

I can switch to escaped double ticks though if you think that helps.

Reply:
I will open another PR with escaped double ticks. And you can approve either/or.

Reply:
@rokroskar here is the PR with escaped double quotes. That works too.

Reply:
Ok so the single of double quotes do not matter because all of these strings passed to `git_cli` are passed to `shlex.split` (https://docs.python.org/3.8/library/shlex.html#shlex.split) before being run with `subprocess.Popen`.

And `shlex.split` does not care whether you use single or double quotes. In both cases it properly figures out what is an argument and what isn't.

```
>>> shlex.split("git config user.name \"Some complicated name\"")
['git', 'config', 'user.name', 'Some complicated name']
>>> shlex.split("git config user.name 'Some complicated name'")
['git', 'config', 'user.name', 'Some complicated name']
```

But not using any quotes like before is definitely wrong:
```
>>> shlex.split("git config user.name Some complicated name")
['git', 'config', 'user.name', 'Some', 'complicated', 'name']
```

Reply:
Ok thanks for checking